packages/isorropia/example/probe_tridiag.cpp

```diff
@@ -77,7 +77,10 @@ int main(int argc, char *argv[])
 
     std::vector<int> NumNz(NumMyElements);
 
-    int i, ierr;
+    int i;
+#ifdef DEBUG
```

Comment:
Is it really `#ifdef DEBUG`?  I thought it was `NDEBUG`, like this:
```c++
#ifndef NDEBUG
int ierr;
#endif 
```

Reply:
I've seen examples of both through out the code. I think there's been _DEBUG as well. So, I've been searching the package and using whichever was previously used in that package.
Isorropia specifically didn't have an example of any of these. So, I used what I had used on the previous package I touched.
I can change it if needed, but as is, it has already passed the GCC 7.2.0 PR test with Werror on.

Reply:
Are there any debug PR test builds?

Reply:
Not currently.
@trilinos/triutils just brought this up as well.
So this will also be needed for #5153 & #5159.
I'll make the changes and file a PR for the one that got merged into Triutils.